content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean.md

```diff
@@ -0,0 +1,120 @@
+---
+title:
+  - 1. Boolean
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Boolean.DT
+  - ib-19.3.Datatype-Primitief-Boolean.DT
+draft: false
+---
+
+> Meer informatie over [[4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean|Boolean]]
+
+## Opdracht 1
+Maak een boolean aan en print het resultaat.
+
+### Specificaties
+- Creëer een `bool` genaamd `IsWaar`.
+- Print de waarde van de `IsWaar uit in de console.
+
+### Verwachte output:
+Als `IsWaar = true`, komt in de console:
+```
+true
+```
+Als `IsWaar = false`, komt in de console:
+```
+false
+```
+
+### Nu jij:
+``` csharp runner
+using System;
+
+Console.WriteLine();
+``` 
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+>
+>bool IsWaar = true;
+>Console.WriteLine(IsWaar);
+> ```
+
```

Comment:
Opmerking: Ik neem aan dat het hier vooral erom gaat dat ze het eerst zelf proberen, voordat ze de mogelijke uitwerking bekijken/kunnen zien? Anders geef je direct het antwoord weg. Kan je zoiets met een [spoiler]  taggen ofzo? Dat ze het zelf moeten openklikken?

Reply:
Dit gebeurd als met de [!info] tag waar het inzit, het is alleen niet te zien binnen de webeditor